src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -53,7 +53,8 @@ public IntakeSubsystem() {
     m_pivotMotor.enableSoftLimit(SoftLimitDirection.kReverse, true);
     m_pivotMotor.setSoftLimit(SoftLimitDirection.kForward, (float) DEPLOY_SETPOINT);
     m_pivotMotor.setSoftLimit(SoftLimitDirection.kReverse,(float) STOW_SETPOINT);
-
+    m_pivotMotor.setSmartCurrentLimit(50);
+    m_pivotMotor.setSmartCurrentLimit(50);
```

Comment:
Why is the current limit being set twice for the same motor? Also, the intake motor needs to be current limited. Also, we should be using a constant from the intake constants class rather than a number for the current limit.